packages/cf.js/src/app/index.ts

```diff
@@ -31,6 +33,7 @@ export class CfAppInstance {
   ) {}
 
   public cfAddress(): H256 {
+    // FIXME: shouldn't have to require abi and bytecode here
```

Comment:
This is a large FIXME and is encompassed with the general direction of the project and is not necessarily just a purely technical fix (requires architectural changes too). For this reason I vote that we remove the FIXMEl